### PR TITLE
Revert "bug: remove all todos (#189)"

### PR DESCRIPTION
**Pull request by tariqajyusuf**
the scourge has been defeated.

---

Re-adds TODOs to trigger the post-submission action.

### CODE REVIEW

File: client/src/components/AppHeader.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ import logo from "../resources/images/logo.svg";
 function AppHeader() {
   const { t } = useTranslation();
 
+  // TODO: Add tests to check rendering and a11y.
```
Comment (kebrahim) [inline client/src/components/AppHeader.jsx:14]: this one is actually obsolete. if you revert the change to this file, in this PR, will it automatically close the corresponding issue? that would be cool. :)

File: client/src/components/NumberQuestion.test.js [hunk 2]
```diff
@@ -271,7 +271,8 @@ describe("NumberQuestion", () => {
       );
 
       expect(getByText("HOUSEHOLD_SIZE.text")).toBeInTheDocument();
-
+      // TODO(tariq): This test is broken because of how we mock out i18n, someone should fix this.
```
Comment (kebrahim) [inline client/src/components/NumberQuestion.test.js:274]: none of the TODOs w/ names attached had corresponding issues created

File: client/src/components/LanguageSwitcher.jsx [hunk 3]
```diff
@@ -12,6 +12,7 @@ import styles from "./LanguageSwitcher.module.css";
 function LanguageSwitcher() {
   const { t } = useTranslation();
 
+  // TODO: Add tests to check rendering, a11y, and i18n.
```
Comment (kebrahim) [inline client/src/components/LanguageSwitcher.jsx:15]: this one is obsolete too

File: client/src/components/NumberQuestion.test.js (hunk 2 above)
Comment (tariqajyusuf) [reply client/src/components/NumberQuestion.test.js:274]: Fixed this by removing the names, will send a chat message about it.
